### DISCUSSION TITLE
Stickers

### DISCUSSION THREAD

**Discussion by zagatta** [elementary/store#88] [Product Ideas]
## Problem
I personally don't have a use for any of the products that the elementary store offers.
Still, I would like to support the project by buying something I can actually use in my daily life.

## Proposal
Offer Stickers. I can slap those on my laptop and support the project with my purchase.
I would recommend stickers for the back of a laptop as well as the SUPER-Key on the keyboard. My personal preference would be very high quality stickers for a higher price than what is offered on redbubble, etc.

## Prior Art
https://system76.com/swag/stickers
https://www.redbubble.com/shop/elementary+os+stickers
https://www.4keyboard.com/gifts-keyboard-stickers/658-elementary-os-sticker.html

**Comment by cassidyjames** [comment]
Thanks for the suggestion! This is probably the top product request, but it's great to get it documented here. Let me provide a bit of context:

1. We use [Printful](https://www.printful.com/) for our merch. This is because they offer excellent quality on-demand products that are easy for us to create and manage. We've been happy with them for the several years we've used them with the previous store. They focus mainly on apparel, but more recently started offering all sorts of things.

2. The new store front-end is basically "just" an API client for Printful, which vastly improves the ease with which we can create more products, adjust pricing, and create variations of products.

3. However, the above mean we're limited to the [product catalog](https://www.printful.com/custom-products) of Printful. They do offer stickers, but from our understanding (including looking at the specs, photos, and reviews), they seem to be lower quality or at least thinner and less wear-resistant than we'd expect for the price.

We've previously used other traditional sticker companies in the past where we ordered hundreds or thousands of stickers in a single order, then manually sent them out in envelopes for orders. The quality was exceptional and we still do carry a small amount of stock of these for Patreon backers and in-person events. However, offering them as one-off purchases became unscalable as we're in the business of making software, not warehousing and shipping physical goods. So we lean on the expertise of Printful plus their on-demand model to provide merch and avoid all of the logistics.

In summary, I think we can offer two solutions:

1. [Back us on Patreon](https://www.patreon.com/elementary)! Backers get a welcome pack of stickers from that aforementioned back stock. The recurring monthly patron model makes this more predictable for us and we generally send these out in batches for new backers each month.

2. We can try Printful stickers. We'd need to design a sticker sheet, price them out, and then order a sample to see how they fair. I think I'm on board with this just for due diligence, and if they're acceptable, we can add them to the store and see if anyone bites. My major concern here is the time and effort it will take to validate them compared to the relatively high price we'd have to offer them at. But maybe enough people would grab them on their own or even as an add-on to an order that it would be worth it.

If option 2 doesn't work out, we _could_ also offer an "upsell" linking to our Patreon for those looking to support us and get stickers. I recall we used to do something similar on our old store.

**Reply by cassidyjames** [reply]
As an update, we've ordered a test print of stickers to help judge their quality. If that goes well, we'll consider adding them to the store.

**Reply by zagatta** [reply]
Awesome! Looking forward to buying them!

**Reply by cassidyjames** [reply]
@zagatta we received the sample order and are still trying to decide if it's worth it. We'd have to sell a 5×5-inch sticker sheet (with 4 or more individual stickers) for around $5 to make it profitable, which is admittedly pretty steep—especially for the quality. I'll update here when we've decided.

**Comment by zagatta** [comment]
I've just ordered the stickers for the elementary developer week. I guess they are from the same source that you would be using for a regular sticker pack?

I'll gladly give feedback here, in terms of my opinion on quality and if I feel it's worth the price, if that would be valuable feedback to you?

**Reply by cassidyjames** [reply]
@zagatta yes, absolutely! We did a one-off sticker set for edw as a sort of live trial of how they were received. We'd love to hear your feedback.

**Reply by zagatta** [reply]
@cassidyjames 

I received the stickers and they definitely could be higher quality, for sure.
The image could be sharper and the colours could be more distinct (currently they look dotted, when you look close).

![IMG_20210707_112452](https://user-images.githubusercontent.com/31660718/124735999-de7de800-df16-11eb-8766-ee552f803a56.jpg)


**That said, I still think they are absolutely worth it.** From my POV this is something where I can support the project one-off (besides regular donations) and receive something back, which doesn't clutter my apartment. 
So I would like to see regular elementary stickers in this quality in the store, for people who don't really need anything but want to show their support. If there the chance to switch to another supplier with better quality in the future, that would be nice, but not essential.

**Reply by cassidyjames** [reply]
@zagatta thanks so much! Yes, that was basically my conclusion as well. We provided vector graphics, so the dotted printing is a result of the printing method itself. It may be exasperated by the mesh gradient which is difficult to reproduce from an RGB file to a CMYK print; simpler graphics with solid colors _may_ come out nicer. But we can work on getting some regular elementary-branded stickers to the store. Worst case they don't sell well, which with the print-on-demand method is not a loss for us.
